### PR TITLE
Add Neo MXNet INF1 notebook

### PR DESCRIPTION
**Pull request by minlu1021**
*Issue #, if available:*

*Description of changes:*
Add MXNet notebook for INF1 target for Neo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    "role = get_execution_role()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "patient-benjamin",
+   "metadata": {},
+   "source": [
+    "# Construct a script for distributed training \n",
+    "The mnist.py script provides all the code we need for training and hosting a SageMaker model."
```
Comment (ngluna) [inline sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb:69]: Add a quick sentence why you are creating a script for distributed training. (as opposed to non-distributed)

**Review by ngluna — COMMENTED** [review]
Looks good. Just one very minor sentence addition and it's ready for merging. :)

File: sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb (hunk 1 above)
Comment (minlu1021) [reply sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb:69]: Thanks for the comment, it is actually not needed. I changed to default(non-distributed) training.

Comment (ngluna) [reply sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb:69]: Awesome. Thanks!

**Review by ngluna — APPROVED** [review]

File: sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb (hunk 1 above)
Comment (minlu1021) [reply sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb:69]: Thank you for the review! Could you help merge this PR?

**Pull request merged**